docs/content/direct/testing.md

```diff
@@ -6,22 +6,20 @@ Make sure all pre-requisites are installed as described in [pre-reqs](pre-reqs.m
 
 The Makefile has a target for running all the unit tests.
 
-```shell

```

Comment:
Do not remove the fencing around fenced code blocks. If you do not know what a fenced code block in Markdown is, please look it up.